test/fixtures/api_services.yml

```diff
@@ -0,0 +1,27 @@
+keith_service:
+  name: Keith Server
+  user: :keith
+  url: http://test.com/models
+  token: abc-secret
+  driver: OpenAI
+
+rob_service:
+  name: Rob Server
+  user: :rob
+  url: http://rob-test.com/models
+  token: rob-secret
+  driver: Anthropic
+
+taylor_anthropic_service:
+  name: Taylor Anth Server
+  user: :taylor
+  url: http://taylor.org/doit
+  token: ta-secret
+  driver: Anthropic
+
+taylor_openai_service:
+  name: Taylor OpenAI Server
+  user: :taylor
+  url: http://taylor.org/doit-open
+  token: open-secret
+  driver: OpenAI
```

Comment:
I think we should put real data in here: the actual data we want for OpenAI and Anthropic. And then we should update seeds.rb to preload this file. This will ensure all new installations get the two core api_services pre-populated.

Reply:
It seems more appropriate to go to lengths to **not** get these values into the prod database. Since they are for actual users. What do I know what real-life users Keith and Rob want in their configurations. (I just made up Taylor.)

However, for the OpenAI/Anthropic api_services values, it seem they should get hard-coded in the User::Registerable.create_initial_assistants method (and db-migration for existing users)

Reply:
Okay, yes, I think you're right. Maybe it was a mistake that I did that for LanguageModel, but anyway, that's out of scope so I can revisit that later.

At the very least, there is some difference between stuff we're creating which is **connected to a User**, like Assistants and API Services.